src/shared/components/palette/dynamic-radio-list/write-dynamic-radio-list-field.component.ts

```diff
@@ -33,7 +33,7 @@ export class WriteDynamicRadioListFieldComponent extends AbstractFieldWriteCompo
     this.dynamicRadioListControl.setValue(this.caseField.value);
   }
 
-  public buildElementId(name: string): string {
-    return `${this.id()}-${name}`;
+  createElementId(name?: string): string {
+    return this.parent ? this.parent.value.id + this.parent.value.value : `${this.id()}-${name}`
```

Comment:
I think that name should not be optional, also rather than just repeating what happens in the abstract class createElementId, why don't you just call it to save repeating code, like so...

```
createElementId(name: string): string {
    return this.parent ? this.parent.value.id + this.parent.value.value : super.createElementId(name);
}
```